plugins/MantisCoreFormatting/MantisCoreFormatting.php

```diff
@@ -158,6 +160,21 @@ function formatted( $p_event, $p_string, $p_multiline = true ) {
 
 		$t_string = $p_string;
 
+		if( null === $s_markdown ) {
+			$s_markdown = plugin_config_get( 'process_markdown' );
+		}
+
+		# Process Markdown and return content, no further text processing.
+		if( ON == $s_markdown ) {
+			if( $p_multiline ) {
+				$t_string = MantisMarkdown::convert_text( $t_string );
+			} else {
+				$t_string = MantisMarkdown::convert_line( $t_string );
```

Comment:
Same as above.

Reply:
fixed and rebased

```php
if( $p_multiline ) {
   return MantisMarkdown::convert_text( $t_string );
} else {
   return MantisMarkdown::convert_line( $t_string );
}
```

Would a ternary here be OK for you?

```php
return $p_multiline
  : MantisMarkdown::convert_text( $t_string )
  ? MantisMarkdown::convert_line( $t_string )
;
```

Reply:
Ternary operator was introduced with PHP 5.3, so @grummbeer that's ok, as starting from version 2.26.0 we support just PHP >= 7.2.5 see https://www.mantisbt.org/bugs/view.php?id=27840
Starting from 2.27.0 it will be PHP >= 7.4,0 see https://www.mantisbt.org/bugs/view.php?id=32808

Reply:
Thanks